src/app/components/vm-list/vm-list.component.html

```diff
@@ -134,6 +134,25 @@
             Sort by Team
           </mat-checkbox>
         </div>
+        <div
+          fxLayout="column"
```

Comment:
Since angular flex-layout is [deprecated](https://blog.angular.io/modern-css-in-angular-layouts-4a259dca9127) now, I think we should try to not add any more uses of it to the codebase. Can you switch this to plain css?

Should be something like

``` css
{
    display: flex;
    flex-direction: column;
    justify-content: space-around;
    align-items: center; 
    align-content: center
}
```